package/yast2-journal.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Aug 28 11:17:02 UTC 2018 - lslezak@suse.cz
+
+- Do not crash when changing the filter as a non-root user
+  (bsc#1089626)
+- 3.2.2
```

Comment:
If I am not wrong, this is the first PR for SP1. Should not the version be 3.3.0 ?

Reply:
It's for SLE12-SP4 so just increasing the patch version, for SLE15-SP1 it will be 4.1.0

Reply:
https://yastgithubio.readthedocs.io/en/latest/versioning/#old-schema